### PR TITLE
Fix isMuted so it doesn't do an extra unit conversion.

### PR DESCRIPTION
**Pull request by picklesrus**
Rename state's muteExpiresAt to muteExpiresAtMs so units are explicit.

### CODE REVIEW

File: src/views/preview/comment/compose-comment.jsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ class ComposeComment extends React.Component {
     }
 
     isMuted () {
-        return this.state.muteExpiresAt * 1000 > Date.now();
```
Comment (picklesrus) [inline src/views/preview/comment/compose-comment.jsx:121]: muteExpiresAt in the state object is already being saved in ms in the constructor and in the comment's post response so I'm pretty sure this one is a bug.

**Review by chrisgarrity — COMMENTED** [review]
It would be good to figure out a test. It seems as though there are two things we're counting on, redis returns expiration in seconds - that's the value in mute_status. Date wants everything in milliseconds.  Could we set up a test to ensure that `state.muteExpiresAtMs == props.mute_status.muteExpiresAt * 1000`.

It would need a second mockStore that set up the mute_status value.

**Comment by picklesrus** [comment]
Good idea. Test added!

**Review by chrisgarrity — APPROVED** [review]
Looks good!

**Comment by picklesrus** [comment]
Tests failed after I rebased because of where now mounted (vs shallow mounted) component was looking for the `store`.  Fixed it up in a separate commit. If that passes, I'll merge this.

**Pull request merged**